jdaviz/cli.py

```diff
@@ -66,6 +65,8 @@ def main(filename, layout='default', browser='default', theme='light', verbosity
     hotreload: bool
         Whether to enable hot-reloading of the UI (for development)
     """
+    import logging  # Local import to avoid possibly messing with JWST pipeline logger.
```

Comment:
Is this here just to make coverage test pass, or is there a dependency between jdaviz and the pipeline? (it's just an out-of-curiosity question).

Reply:
This import is needed for `logging.getLogger('tornado.access').disabled = True` on L99 below. I moved it from global to local import in fear `stpipe` might pick it up.